conda_recipe/meta.yaml

```diff
@@ -36,6 +38,10 @@ requirements:
     - scikit-learn
     - sqlalchemy
     - ujson
+    - requests
+    - requests-mock
+    - pytest
```

Comment:
Out of curiousy, why this if there's a separate section for test requirements?

Reply:
conda build fails in the test part if these packages are in build requirements only.